pkg/procsyms/procsysms.go

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Tetragon
+
+package procsyms
+
+import (
+	"fmt"
+	"sync"
+
+	lru "github.com/hashicorp/golang-lru/v2"
+)
+
+var (
```

Comment:
this is not use in generic code, I think it should go to _linux.go object

Reply:
This is used in the function signature of `GetFnSymbol(pid int, addr uint64) (*FnSym, error)` which is a package export and linked from various points.

Reply:
As per what we have agreed, I think we need to minimize the code added to platform specific files and maximize code added to platform neutral files. In that light, if a var is used only in Linux, but builds on Windows, shouldn't it stay in common code ?

Reply:
hum, but both cache and setCache vars are in linux code only right? I don't why to have them in common code

Reply:
They were originally in `procsyms.go` and I moved only Linux specific code - code that wouldn't compile on Windows, into `procsysms_linux.go`. 
I guess there are more than one right answers for this.